seacrowd/sea_datasets/pho_ner_covid/pho_ner_covid.py

```diff
@@ -0,0 +1,203 @@
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = []
+
+    for subset_id in _SUBSETS:
+        BUILDER_CONFIGS.append(
+            SEACrowdConfig(
+                name=f"{subset_id}_source",
+                version=SOURCE_VERSION,
+                description=f"{_DATASETNAME} source schema",
+                schema="source",
+                subset_id=subset_id,
+            )
+        )
+
+        seacrowd_schema_config: list[SEACrowdConfig] = []
+
+        for seacrowd_schema in _SUPPORTED_SCHEMA_STRINGS:
+
+            seacrowd_schema_config.append(
+                SEACrowdConfig(
+                    name=f"{subset_id}_{seacrowd_schema}",
+                    version=SEACROWD_VERSION,
+                    description=f"{_DATASETNAME} {seacrowd_schema} schema",
+                    schema=f"{seacrowd_schema}",
+                    subset_id=subset_id,
+                )
+            )
+
+        BUILDER_CONFIGS.extend(seacrowd_schema_config)
+
+    DEFAULT_CONFIG_NAME = f"{_SUBSETS[0]}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "words": datasets.Sequence(datasets.Value("string")),
+                    "tags": datasets.Sequence(datasets.ClassLabel(names=_TAGS)),
+                }
+            )
+
+        elif self.config.schema == f"seacrowd_{str(TASK_TO_SCHEMA[Tasks.NAMED_ENTITY_RECOGNITION]).lower()}":
+            features = schemas.seq_label_features(label_names=_TAGS)
+
+        else:
+            raise ValueError(f"Invalid config: {self.config.name}")
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        split_generators = []
+
+        for split in _SPLITS:
+            path = dl_manager.download_and_extract(_URLS[_DATASETNAME][self.config.subset_id][split])
+
+            split_generators.append(
+                datasets.SplitGenerator(
+                    name=split,
+                    gen_kwargs={
+                        "path": path,
+                    },
+                )
+            )
+
+        return split_generators
+
+    def _generate_examples(self, path: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+
+        idx = 0
+        df = pd.read_json(path, lines=True)
+
+        if self.config.schema == "source":
+            for _, row in df.iterrows():
+                yield idx, row.to_dict()
+                idx += 1
+
+        elif self.config.schema == f"seacrowd_{str(TASK_TO_SCHEMA[Tasks.NAMED_ENTITY_RECOGNITION]).lower()}":
```

Comment:
Since you've reused the `str(TASK_TO_SCHEMA...`, perhaps you can assign this to a `schema` variable instead?